docs/01-installing.md

```diff
@@ -1,3 +1,9 @@
+# Prerequisites
+
+* There were some breaking changes in the development. To rule out incompatibilities make sure you have installed the latest version of [``Noevim 0.5``](https://github.com/neovim/neovim/releases/tag/v0.5.0).
```

Comment:
Chris wants the documentation to be as minimal as possible.  Remove unneeded words.  The above can be shortened to `make sure you have installed the latest version of [``Noevim 0.5``]`

Reply:
Done, please check